rke/k8s_version_info.go

```diff
@@ -87,10 +87,14 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MaxRancherVersion: "2.2.99",
 			MaxRKEVersion: "0.2.99",
 		},
+		"v1.15.12-rancher2-2": {
```

Comment:
This change isn't needed if we revert this version back to `v1.15.12-rancher1-1`